data_collection/web/guides.rst

```diff
@@ -0,0 +1,717 @@
+
+.. code-block:: javascript
+
+  _paq.push(["disableLinkTracking"]);
+
+Disabling link tracking with CSS classes
+""""""""""""""""""""""""""""""""""""""""
+
+You can mark links that you do not with to track with CSS classes. JS Tracking Client will ignore such links and won't track them.
+
+.. code-block:: javascript
+
+  _paq.push(["setIgnoreClasses", "do-not-track"]);
+  _paq.push(["trackPageView"]);
+
+or a list of classes:
+
+.. code-block:: javascript
+
+  _paq.push(["setIgnoreClasses", ["dont-track-this", "this-either", "nor-this"]]);
+  _paq.push(["trackPageView"]);
+
+and using that class in HTML code:
+
+.. code-block:: html
+
+  <a href="https://piwik.pro/document.pdf" class="dont-track-this">A document, that should not be tracked.</a>
+
+.. _guide_tracking_link_clicks_on_pages_with_dynamically_generated_content:
+
+Tracking link clicks on pages with dynamically generated content
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+When you want to track clicks on the links, which are dynamically added to the HTML document, you have to call :ref:`enableLinkTracking<jtc-api-enableLinkTracking>` every time when the new links are added to the document.
+
+For fully static pages calling :ref:`enableLinkTracking<jtc-api-enableLinkTracking>` once is enough, because each call adds listeners only for those links, which are currently present in the HTML document. So if you add new links to the document and you want to track them, you have to call :ref:`enableLinkTracking<jtc-api-enableLinkTracking>` each time that happens.
+
+.. code-block:: javascript
+
+    // Add click listeners to new links
+    _paq.push(["enableLinkTracking"]);
+
+.. note::
+
+  You don't have to call :ref:`enableLinkTracking<jtc-api-enableLinkTracking>` if you are tracking links with inline JavaScript (with :ref:`trackLink<marking-outlinks-inline-calls>`).
+
+Goal tracking
+-------------
+
+At this point we have tracked lots of various typose of events. We have regular page views, we have downloads, outlinks, custom events and others. Above them all there's one more event type we can track: a conversion.
+And goal tracking is about tracking conversions. If you can point out parts of your website/application more important from your bisness perspective, you could :ref:`define those parts as goals<https://help.piwik.pro/support/analytics-new/goals/>`.
+Visiting a specific landing page, submitting a contact form, downloading a PDF file with your product manual - these are popular examples of goal definitions. You can even define a goal based on the custom event you are tracking.
+
+Once the goal is defined, every time a tracked event fits its definition, an additional conversion event will be created. We call this procedure an "automatic conversion".
+
+Alternatively, you can trigger a goal manually with the used of
+
+.. code-block:: javascript
+
+    // force conversion of the goal with ID 17
+    _paq.push(["trackGoal", 17]);
+
+.. note::
```

Comment:
This note doesn't contain any text. Should we remove it?

Reply:
We can remove it.

Reply:
Done